types.d.ts

```diff
@@ -8,3 +9,14 @@ declare global {
 		}
 	}
 }
+
+declare namespace expressRequestId {
+	interface Options {
+		setHeader?: boolean | undefined;
+		headerName?: string | undefined;
+		generator?: (request: Request) => string | undefined;
```

Comment:
```suggestion
		generator?: ((request: Request) => string) | undefined;
```
func or undefined, tsc resolves this wronlgy without brackets